youtube_dl/extractor/twitcasting.py

```diff
@@ -2,13 +2,16 @@
 from __future__ import unicode_literals
 
 from .common import InfoExtractor
+from ..utils import (
```

Comment:
```suggestion
from ..utils import urlencode_postdata
```

Reply:
Thank you for your reviewing.
Fixed import format (db60fba)